Rakefile

```diff
@@ -26,6 +26,7 @@ RSpec::Core::RakeTask.new(:spec) do |spec|
   spec.rspec_opts = def_spec_options
   spec.rspec_opts << ' --tag ~integration'
   spec.rspec_opts << ' --tag ~system'
+  spec.rspec_opts << ' --tag ~integration_i3s'
```

Comment:
@tmiotto , I tried to run it with two tags, this way: `--tag integração --tag i3s`, but always run the other integration tests, not only for the i3s. It may be that I do not know the correct form. Do you have any suggestion?